src/lib/y2storage/storage_env.rb

```diff
@@ -89,6 +92,15 @@ def luks2_available?
       active?(ENV_LUKS2_AVAILABLE, default: false)
     end
 
+    # Whether YaST should reuse existing LVM
+    #
+    # see jsc#PED-6407 or jsc#IBM-1315
+    #
+    # @return [Boolean, nil] boolean as explicitly set by user, nil if user set nothing
+    def reuse_lvm?
+      active?(ENV_REUSE_LVM, default: nil)
```

Comment:
One more concern. I had totally overlooked this.

A method with a name ending with a question mark but not always returning a boolean (`nil` is possible) looks totally unexpected and anti-ruby to me.

Reply:
ok ... I'm not ruby expert. I'll rename that